.github/workflows/docker-build.yml

```diff
@@ -92,12 +92,16 @@ jobs:
       - name: Delete Temporary Pod
         run: kubectl delete pod -n ${{ env.EKS_NAMESPACE }} temp-pod-${{ steps.vars.outputs.sha_short }}
 
+      - name: Set cl-python HPA maxReplicas
+        run: kubectl patch hpa cl-python -n ${{ env.EKS_NAMESPACE }} --type=merge -p '{"spec":{"maxReplicas":10}}'
```

Comment:
I don't love this. One day we may have 12 because we're under load, and it'd be bad to reduce it to 10. 

Could we grab the current number from the cluster and set it as the max, run the deploy, then reset to 32 instead?